src/repository/streaming-fetcher.ts

```diff
@@ -178,8 +178,7 @@ export class StreamingFetcher extends EventEmitter implements FetcherInterface {
         instanceId: this.instanceId,
         etag: undefined,
         contentType: undefined,
-        custom: this.headers,
-        specVersionSupported: '5.2.0',
```

Comment:
This was always out of sync so I added client-spec-version.ts helper (please review it)